deployment.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Deployment
+layout: recommendation
+expires: 2017-12-01
+---
+
+Teams should create standalone artifacts which can:
+
+- have automated tests run against them to ensure they work as expected
+- be deployed to a pre-production environment for manual testing
+- be promoted to a production environment
+
+The artifact should contain the running application, including
+any dependencies.
```

Comment:
Can we have something that covers being able to cryptographically prove it's the same between environments please?